watch.js

```diff
@@ -55,30 +59,42 @@ FirebaseWatcher.prototype.connect = function() {
 
 		if (!isNaN(msg)) {
 			if (self.state == IGNORE_NEXT_STREAM) {
-				self._rootReceived = Date.now();
-				self.emit('serverReady', self._rootReceived - self._rootRequested)
+				self.log('S: IGNORED_STREAM');
```

Comment:
They're all behind a `debug` option, so they're all no-ops by default.

I found myself constantly adding these in and then deleting them every time I've needed to fix something, so I wanted to make life easier by having this code in permanently.